dbt/adapters/databricks/impl.py

```diff
@@ -59,6 +60,19 @@ class DatabricksAdapter(SparkAdapter):
 
     AdapterSpecificConfigs = DatabricksConfig
 
+    @available.parse(lambda *a, **k: 0)
```

Comment:
Just curious, why do we need this `@available.parse`?

Reply:
Without that, the function will be executed even at parse time that causes `InvalidConnectionException` because connections are not available at parse time.

> A decorator factory to indicate that a method on the adapter will be
        exposed to the database wrapper, and will be stubbed out at parse time
        with the given function.